components/heap/third_party/dartino/cmpctmalloc.c

```diff
@@ -145,8 +145,12 @@ static int first_allocations = true;
 #define ROUND_UP(x, alignment) (((x) + (alignment) - 1) & ~((alignment) - 1))
 #define ROUND_DOWN(x, alignment) ((x) & ~((alignment) - 1))
 #define IS_ALIGNED(x, alignment) (((x) & ((alignment) - 1)) == 0)
+#ifndef MIN
```

Comment:
Maybe undefine them instead? Do you want to use the MIN and MAX from somewhere else?